docs/a320-simvars-table.md

```diff
@@ -0,0 +1,550 @@
+| A32NX_VSPEEDS_F               | Knots   | F-Speed (approach)                                                                             |
+| A32NX_VSPEEDS_GD              | Knots   | Green Dot speed (clean config or O) (is mach corrected)                                        |
+| A32NX_VSPEEDS_LANDING_CONF3   | Boolean | True if FLAPS 3 is selected in perf page                                                       |
+| A32NX_VSPEEDS_S               | Knots   | S-Speed (approach)                                                                             |
+| A32NX_VSPEEDS_TO_CONF         | Number  | Flaps config for TakeOff, 1, 2 or 3                                                            |
+| A32NX_VSPEEDS_V2              | Knots   | TakeOff V2 Speed calculated based on A32NX_VSPEEDS_TO_CONF config                              |
+| A32NX_VSPEEDS_VAPP            | Knots   | vapp calculated for config full whether A32NX_VSPEEDS_LANDING_CONF3 or not (is mach corrected) |
+| A32NX_VSPEEDS_VLS             | Knots   | Current config minimum selectable speed (is mach corrected)                                    |
+| A32NX_VSPEEDS_VLS_APP         | Knots   | VLS calculated for config full whether A32NX_VSPEEDS_LANDING_CONF3 or not (is mach corrected)  |
+| A32NX_VSPEEDS_VS              | Knots   | Current config stall speed (is mach corrected)                                                 |
+
+## Throttle Mapping System
+
+| Name                                                                       | Type             | Description                                                                                                                                                                                                |
+|:---------------------------------------------------------------------------|:-----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| A32NX_AUTOTHRUST_TLA:{index}                                               | Number (degrees) | Indicates the TLA of the throttle lever {index}, first throttle lever has index 1 <br/>Position = Value <br/>REVERSE = -20 <br/>REV_IDLE = -6 <br/>IDLE = 0 <br/>CLB = 25 <br/>FLX/MCT = 35 <br/>TOGA = 45 |
+| A32NX_THROTTLE_MAPPING_A32NX_<br/>THROTTLE_MAPPING_{name1}_{name2}:{index} | Number           | Indicates the low or high value to latch into the given detent<br/>Range is from -1 to 1<br/>{name1}<br/>REVERSE<br/>REVERSE_IDLE<br/>IDLE<br/>CLIMB<br/>FLEXMCT<br/>TOGA<br/>{name2}<br/>LOW<br/>HIGH     |
+| A32NX_THROTTLE_MAPPING_INPUT:{index}                                       | Number           | Indicates the raw input values for throttle axis {index}, first axis has index 1<br/>Range is from -1 to 1                                                                                                 |
```

Comment:
**question**: Should these also be split?

Reply:
With split you mean expanding without {index} or moving the A32NX_AUTOTHRUST_TLA to Autothrust?
I will expand {name} into lines. I think the {index} is fine, ok?